phf_macros/src/lib.rs

```diff
@@ -225,34 +227,40 @@ fn parse_set(cx: &mut ExtCtxt, tts: &[TokenTree]) -> Option<Vec<Entry>> {
     Some(entries)
 }
 
-fn parse_key(cx: &mut ExtCtxt, e: &Expr) -> Option<Key> {
+fn parse_key(cx: &mut ExtCtxt, e: P<Expr>) -> Option<(Key, P<Expr>)> {
```

Comment:
Thanks! I'm about to be stuck on a plane but I'll merge when it lands.
On Sat, Jan 30, 2016 at 6:33 PM Anthony Ramine <notifications@github.com>
wrote:

> In phf_macros/src/lib.rs
> <https://github.com/sfackler/rust-phf/pull/77#discussion_r51354267>:
>
> > @@ -225,34 +227,40 @@ fn parse_set(cx: &mut ExtCtxt, tts: &[TokenTree]) -> Option<Vec<Entry>> {
> >      Some(entries)
> >  }
> >
> > -fn parse_key(cx: &mut ExtCtxt, e: &Expr) -> Option<Key> {
> > +fn parse_key(cx: &mut ExtCtxt, e: P<Expr>) -> Option<(Key, P<Expr>)> {
>
> Fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sfackler/rust-phf/pull/77/files#r51354267>.
>